pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 2.0.1
+title: Release notes for 2.0.1
+menuWeight: 2
+excerpt: Release notes for DC/OS 2.0.1
+---
+DC/OS 2.0.1 was released on 22 November 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/2.0.1/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/2.0.1/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+
+# Release summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# Fixed and Improved Issues in DC/OS 2.0.1
+<!-- The issues that have been fixed and improved in DC/OS 2.0.1 are grouped by feature, functional area, or component.  -->
```

Comment:
We would also need to add the release note for mesos memory leak issue. cc @greggomann

Reply:
Can one of you provide the JIRA ticket number?

Reply:
https://jira.mesosphere.com/browse/DCOS-61228

Reply:
ty. resolved.